src/UI/components/map/layers/layerControl.tsx

```diff
@@ -101,7 +101,7 @@ export const LayerControl = ({
           >
             <CloseIcon />
           </IconButton>
-        )}
+        )} */}
```

Comment:
Commented out code?

Reply:
This is because it's unclear if Marianne will want to put the picker back in for base layers after seeing the change. I've added a comment to explain.